iep-module-userservice/src/main/java/com/netflix/iep/userservice/CompositeUserService.java

```diff
@@ -56,10 +59,12 @@
   }
 
   @Override public String toValidEmail(String email) {
-    for (UserService service : services) {
-      String v = service.toValidEmail(email);
-      if (v != null) {
-        return v;
+    if(email != null) {
+      for (UserService service : services) {
+        String v = service.toValidEmail(email.toLowerCase());
```

Comment:
Specify a locale for the toLowerCase conversion, otherwise it is system dependent. `Locale.US` should work fine for this purpose.